From: =?utf-8?q?Florian_M=C3=BCllner?= <fmuellner@gnome.org>
Date: Wed, 6 Mar 2019 23:31:43 +0100
Subject: shadow-factory: Update shadow of attached modals

When commit 91c6a144da synced shadows with Adwaita, it removed the
shadow completely from attached modal dialogs. However Adwaita uses
the same shadow for all dialogs (modal or not), so do the same here.

https://gitlab.gnome.org/GNOME/mutter/issues/490

Origin: upstream, commit:a87e87a05fb29e8e9aad5df5c7752b6db20535e
Applied-Upstream: 3.32.1
---
 src/compositor/meta-shadow-factory.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/compositor/meta-shadow-factory.c b/src/compositor/meta-shadow-factory.c
index 9fdecac..0eb14c9 100644
--- a/src/compositor/meta-shadow-factory.c
+++ b/src/compositor/meta-shadow-factory.c
@@ -128,7 +128,7 @@ MetaShadowClassInfo default_shadow_classes[] = {
   { "popup-menu",    { 1, -1, 0, 0, 128 }, { 1, -1, 0, 0, 128 } },
   { "dropdown-menu", { 1, -1, 0, 0, 128 }, { 1, -1, 0, 0, 128 } },
 
-  { "attached",      { 0, -1, 0, 0, 0 }, { 0, -1, 0, 0, 0 } }
+  { "attached",      { 10, -1, 0, 3, 128 }, { 8, -1, 0, 2, 64 } }
 };
 
 G_DEFINE_TYPE (MetaShadowFactory, meta_shadow_factory, G_TYPE_OBJECT);
